src/constants.js

```diff
@@ -0,0 +1,12 @@
+export const pageFadeTimeout = 400;
```

Comment:
Having a separate constants file to control aspects of the appearance is going to be super-helpful.